### ISSUE TITLE
could not install the plugin

### ISSUE THREAD

**Issue by tboulogne** [taigaio/taiga-contrib-gogs#1]
Hello,

When i try 

```
python manage.py migrate taiga_contrib_gogs
```

```
 Apply all migrations: taiga_contrib_gogs
Running migrations:
  Applying taiga_contrib_gogs.0001_initial...Traceback (most recent call last):
  File "/home/taiga/.virtualenvs/taiga/lib/python3.4/site-packages/django/db/backends/utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
psycopg2.ProgrammingError: column "github_id" of relation "users_user" does not exist
LINE 1: ...orize_tags", "token", "email_token", "new_email", "github_id...
                                                             ^
```

any tips to solve this ?

Thanks for your help.

Regards,

**Comment by bameda** [comment]
Try with

```
python manage.py migrate
```

**Comment by tboulogne** [comment]
thanks for answer.
no effect, same error :-(

**Comment by superalex** [comment]
Could you execute a "python manage.py migrate" to check if you have more pending migrations please?

**Comment by donald-slagle** [comment]
I am getting the same error on a clean install of Taiga. I am running version 1.6.0 and have no pending migrations.

I went back and dropped the db then ran migrations and that worked. It seems the problem is that Taiga removed the github_id column from the users table with this: https://github.com/taigaio/taiga-back/blob/master/taiga/users/migrations/0007_auto_20150209_1611.py migration. The github id property isn't on the Taiga user model either but django's orm is still trying to run an insert query with that column when trying to migrate this plugin. So unless you are installing this plugin before you run migrations then it won't work.

**Comment by jespino** [comment]
I'm going to try your path (install taiga-back, generate the database, install the plugin, try to migrate).

Anyway this is really strange, the migration that you say must be applied when you try to migrate the database to apply the plugin, and don't need to be reaplied.

Thanks for your feedback.

**Comment by jespino** [comment]
I found the error, is a dependency error, we need to depend on 0007 migration from users, I will fix this in the next commit.

Thank you for your feedback and your patience. :smile:

**Comment by superalex** [comment]
Thanks @jespino for finding the solution! :+1:

**Comment by jespino** [comment]
Done! and released the new version in PyPi (0.0.3).
